changelogs/unreleased/bump-libcurl-to-8.10.1.md

```diff
@@ -1,3 +1,3 @@
 ## bugfix/build
 
-* Updated libcurl to version 8.10.1 (gh-10576).
+* Updated libcurl to version 8.10.1+patches (gh-10576).
```

Comment:
`+patches` hardly adds meaning.
If this change of build options affects users somehow, we better write the full sentence:
`Updated libcurl and patched its build options`
If it's just internal kitchen, then we better leave the previous text to avoid questions and confusion.

Reply:
Probably it's better to specify an exact version (`curl-8_10_1-241-g461ce6c61`) to avoid confusion with `+ patches` in changelog entry.

Reply:
@p7nov Fair point!

@Col-Waltz, @ligurio Thanks for the suggestions! I think that an explicit `git describe` version is the best option. Updated this way.

Reply:
Thanks!